src/main/java/Duke.java

```diff
@@ -41,45 +41,42 @@ public static void main(String[] args) {
                         int index = Integer.parseInt(input.split(" ")[1]);
                         tasks.get(index - 1).setDone(true);
                         System.out.println("Nice! I've marked this task as done: ");
-                        //System.out.println(tasks.get(index - 1).getSymbol() + " " + tasks.get(index - 1).getTaskDescription());
                         System.out.println(tasks.get(index - 1).toString());
                         break;
                     case "todo":
-                        ToDos t = new ToDos(input.substring(5,input.length()));
+                        ToDos t = new ToDos(input.substring(5, input.length()));
                         tasks.add(t);
-                        System.out.println("Got it. I've added this task:");
-                        System.out.println(t.toString());
-                        System.out.println("Now you have "+ tasks.size()+" tasks in the list.");
+                        printTask(t, tasks);
                         break;
                     case "deadline":
                         String[] deadlineContent = input.split(" /by ");
-                        Deadlines d =new Deadlines(deadlineContent[0].substring(9,deadlineContent[0].length()),deadlineContent[1]);
+                        Deadlines d = new Deadlines(deadlineContent[0].substring(9, deadlineContent[0].length()), deadlineContent[1]);
                         tasks.add(d);
-                        System.out.println("Got it. I've added this task:");
-                        System.out.println(d.toString());
-                        System.out.println("Now you have "+ tasks.size()+" tasks in the list.");
+                        printTask(d, tasks);
                         break;
                     case "event":
-                        String[]  eventContent = input.split(" /at ");
-                        Events e =new Events(eventContent[0].substring(6,eventContent[0].length()),eventContent[1]);
+                        String[] eventContent = input.split(" /at ");
+                        Events e = new Events(eventContent[0].substring(6, eventContent[0].length()), eventContent[1]);
                         tasks.add(e);
-                        System.out.println("Got it. I've added this task:");
-                        System.out.println(e.toString());
-                        System.out.println("Now you have "+ tasks.size()+" tasks in the list.");
+                        printTask(e, tasks);
                         break;
                     default:
                         System.out.println("Command is unknown,please re-try.");
                 }
-            }catch(Exception e){
+            } catch (Exception e) {
                 System.out.println("Internal error occurs,please re-enter the command.");
```

Comment:
By catching this exception, will you be able to find out what's wrong or which function is causing the error? 
And should it continue when it hit an error?